### PR TITLE
[PAY-949][PAY-950][PAY-958][PAY-957] Gated content QA fixes

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

- Dismiss locked content modal on signup redirect
- Dismiss locked content modal on send tip click
- Add ellipsis menu to gated track tiles and fix click behavior
- Update follow and send tip analytics
- and more fixes...

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local dapp vs stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

existing

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

File: packages/common/src/store/premium-content/sagas.ts [hunk 1]
```diff
@@ -335,35 +333,38 @@ const PREMIUM_TRACK_POLL_FREQUENCY = 1000
...
-      trackPageActions.fetchTrack(
-        null /* trackId */,
-        slug,
-        handle,
-        false /* canBeUnlisted */,
-        true /* forceRetrieveFromSource */,
-        false /* withRemixes */,
-        true /* skipSideEffects */
-      )
-    )
```
Comment (dharit-tan) [inline packages/common/src/store/premium-content/sagas.ts:366]: nice

File: packages/common/src/store/premium-content/sagas.ts [hunk 2]
```diff
@@ -449,17 +454,6 @@ function* handleTipGatedTracks(
   yield* call(updateGatedTracks, action.payload.userId, 'tip')
 }
 
-function* refreshPremiumTrackAccess(
```
Comment (dharit-tan) [inline packages/common/src/store/premium-content/sagas.ts:452]: How come we no longer need to poll?

File: packages/web/src/components/track/desktop/BottomRow.tsx [hunk 3]
```diff
@@ -110,15 +110,20 @@ export const BottomRow = ({
...
+            <LoadingSpinner className={styles.spinner} />
+            {messages.unlocking}
+          </div>
+        ) : (
+          <div className={styles.premiumContent}>
+            <IconLock />
+            {messages.locked}
+          </div>
+        )}
+        {!isLoading && <div>{rightActions}</div>}
```
Comment (dharit-tan) [inline packages/web/src/components/track/desktop/BottomRow.tsx:126]: Nit: Per discussion last week, do we want to avoid `&&`? cc @dylanjeffers ?

**Review by dharit-tan — APPROVED** [review]

File: packages/common/src/store/premium-content/sagas.ts (hunk 2 above)
Comment (sddioulde) [reply packages/common/src/store/premium-content/sagas.ts:452]: we do still poll, just not polling via this function anymore

File: packages/web/src/components/track/desktop/BottomRow.tsx (hunk 3 above)
Comment (sddioulde) [reply packages/web/src/components/track/desktop/BottomRow.tsx:126]: yes we do. thx

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-gated-qa

**Pull request merged**